### ISSUE TITLE
css image style not repected for static 

### ISSUE THREAD

**Issue by yakir12** [webpro/reveal-md#214]
I'm trying to get rid of the image borders by supplying a css file `style.css` which contains:
```html
.reveal section img {
    background:none; 
    border:none; 
    box-shadow:none; }
```
and this command: `reveal-md presentation.md -w --css style.css`
That works, for instance:
![screenshot_2018-09-21_13 47 12](https://user-images.githubusercontent.com/3281957/45879486-fd7e5000-bda4-11e8-9ecb-b7b52a716664.png)

But if I export it to a static site (for later hosting on github), with the command (`whyjulia` is the repo I push to git):
`reveal-md presentation.md --css style.css --static whyjulia`
Then I get this:
![screenshot_2018-09-21_13 51 51](https://user-images.githubusercontent.com/3281957/45879631-839a9680-bda5-11e8-9057-2a1bc87ba418.png)
Notice the border around the image. 

Any way I can get rid of those borders? 
Everything is [here](https://github.com/yakir12/whyjulia) if that helps.

Thanks!

**Comment by webpro** [comment]
Hi, @yakir12! With a static export, custom assets need to be copied over as well using `--static-dirs`: https://github.com/webpro/reveal-md#static-website

**Comment by yakir12** [comment]
But no assets were used. My images are url links, such as:
```md
![](https://raw.githubusercontent.com/JuliaGraphics/julia-logo-graphics/master/images/three-balls.svg?sanitize=true) <!-- .element height="10%" width="10%" -->
```
Not sure what to supply `--static-dirs` with..?

**Comment by webpro** [comment]
Ah, my apologies. You mentioned a custom `style.css` file. But now I see that reveal-md is already smart enough to copy that over.

So, is the stylesheet loaded by the browser? I'm seeing a `<link rel="stylesheet" href="./_assets/style.css">` as the last stylesheet, so that's looking fine. Does the `.reveal section img` selector actually match with the HTML that is eventually rendered?

**Comment by webpro** [comment]
OK, I actually cloned your repo. Started a static server from the root. It displays the image without a border, and I can see the styles from `style.css` are loaded and applied (tried in two different browsers).

Also, even opening the deck with `index.html` from the file system works/looks fine.

You mentioned "live reload". Can you elaborate?

**Comment by yakir12** [comment]
1. Thank you for the effort.
2. I do not understand why it worked for you and not me #privilegediscrimination (joking).
3. raw opening the `index.html` worked for me too!!
4. live load: just the normal `reveal-md presentation.md -w`, sorry for the terminology, I was just trying to use [this](https://github.com/webpro/reveal-md#live-reload) terminology.

**Comment by webpro** [comment]
I'm going to close this ticket. Don't think I can/should debug this particular issue on your end. Feel free to (re)open and discuss. Good luck!

**Comment by yakir12** [comment]
I understand the conundrum, but:
* I ran this `reveal-md presentation.md --css style.css --static whyjulia`
* I `git push`ed (after a `add` and `commit`) the content of `whyjulia` to github
* I changed the setting for that github repo so it'd behave as a website
* and the content is accessible on https://yakir12.github.io/whyjulia

If you go to https://yakir12.github.io/whyjulia/#/ or to https://yakir12.github.io/whyjulia/#/10 you'll see the image borders. 

**WAIT!!!**
I got these errors when running the reveal command:
```bash
$ reveal-md presentation.md --css style.css --static whyjulia
Reveal-server started at http://localhost:1948
[Error while generating featured slide snapshot for "undefined"]
Error: Failed to launch chrome!
[0921/211738.421559:ERROR:icu_util.cc(133)] Invalid file descriptor to ICU data received.
[0921/211738.421662:FATAL:content_main_delegate.cc(53)] Check failed: false. 
#0 0x563d47c36b0c base::debug::StackTrace::StackTrace()
#1 0x563d47c4f780 logging::LogMessage::~LogMessage()
#2 0x563d45fd8d23 content::ContentMainDelegate::TerminateForFatalInitializationError()
#3 0x563d47971deb content::ContentMainRunnerImpl::Initialize()
yakir@debian:~/documents/formalia/Presentations/julia2018$ reveal-md presentation.md --css style.css --static whyjulia
Reveal-server started at http://localhost:1948

[Error while generating featured slide snapshot for "undefined"]
Error: Failed to launch chrome!
[0921/211954.280983:ERROR:icu_util.cc(133)] Invalid file descriptor to ICU data received.
[0921/211954.281082:FATAL:content_main_delegate.cc(53)] Check failed: false. 
#0 0x56075c6b2b0c base::debug::StackTrace::StackTrace()
#1 0x56075c6cb780 logging::LogMessage::~LogMessage()
#2 0x56075aa54d23 content::ContentMainDelegate::TerminateForFatalInitializationError()
#3 0x56075c3eddeb content::ContentMainRunnerImpl::Initialize()
#4 0x56075c3f7b72 service_manager::Main()
#5 0x56075c3ecf14 content::ContentMain()
#6 0x5607604fc9b9 headless::(anonymous namespace)::RunContentMain()
#7 0x5607604fca42 headless::HeadlessBrowserMain()
#8 0x56075c3f4f9d headless::HeadlessShellMain()
#9 0x56075aa531ac ChromeMain
#10 0x7f24042d12e1 __libc_start_main
#11 0x56075aa5302a _start



TROUBLESHOOTING: https://github.com/GoogleChrome/puppeteer/blob/master/docs/troubleshooting.md

Error: Failed to launch chrome!
[0921/211954.280983:ERROR:icu_util.cc(133)] Invalid file descriptor to ICU data received.
[0921/211954.281082:FATAL:content_main_delegate.cc(53)] Check failed: false. 
#0 0x56075c6b2b0c base::debug::StackTrace::StackTrace()
#1 0x56075c6cb780 logging::LogMessage::~LogMessage()
#2 0x56075aa54d23 content::ContentMainDelegate::TerminateForFatalInitializationError()
#3 0x56075c3eddeb content::ContentMainRunnerImpl::Initialize()
#4 0x56075c3f7b72 service_manager::Main()
#5 0x56075c3ecf14 content::ContentMain()
#6 0x5607604fc9b9 headless::(anonymous namespace)::RunContentMain()
#7 0x5607604fca42 headless::HeadlessBrowserMain()
#8 0x56075c3f4f9d headless::HeadlessShellMain()
#9 0x56075aa531ac ChromeMain
#10 0x7f24042d12e1 __libc_start_main
#11 0x56075aa5302a _start



TROUBLESHOOTING: https://github.com/GoogleChrome/puppeteer/blob/master/docs/troubleshooting.md

    at onClose (/usr/lib/node_modules/reveal-md/node_modules/puppeteer/node6/lib/Launcher.js:307:14)
    at Interface.helper.addEventListener (/usr/lib/node_modules/reveal-md/node_modules/puppeteer/node6/lib/Launcher.js:296:50)
    at emitNone (events.js:91:20)
    at Interface.emit (events.js:185:7)
    at Interface.close (readline.js:319:8)
    at Socket.onend (readline.js:108:10)
    at emitNone (events.js:91:20)
    at Socket.emit (events.js:185:7)
    at endReadableNT (_stream_readable.js:978:12)
    at _combinedTickCallback (internal/process/next_tick.js:80:11)
    at process._tickCallback (internal/process/next_tick.js:104:9)
```

**Comment by webpro** [comment]
The error regarding "featured slide snapshot" shouldn't happen, as the related argument(s) weren't provided. I've just released [v2.1.2](https://github.com/webpro/reveal-md/releases/tag/2.1.2) that should not even try to use puppeteer in this case.

**Comment by yakir12** [comment]
Wow, thanks, but alas the frames are still [there](https://yakir12.github.io/whyjulia/#/). It's really minor and I've given the presentation a few hours ago, so no biggy.
Thanks for the wonderful work!!!
